app/src/main/java/com/criteo/pubsdk_android/integration/IntegrationSelectorActivity.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ *    Copyright 2020 Criteo
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.criteo.pubsdk_android.integration
+
+import android.os.Bundle
+import android.widget.Button
+import android.widget.CheckBox
+import android.widget.RadioButton
+import androidx.appcompat.app.AppCompatActivity
+import com.criteo.publisher.DependencyProvider
+import com.criteo.publisher.integration.IntegrationDetector
+import com.criteo.pubsdk_android.R
+import com.criteo.pubsdk_android.mock.MockedDependencyProvider
+
+class IntegrationSelectorActivity : AppCompatActivity() {
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_integration_selector)
+
+    findViewById<Button>(R.id.btnSaveIntegrationDetectionValues).setOnClickListener {
+      commitConfiguration()
+    }
+  }
+
+  private fun commitConfiguration() {
+    val verbose = findViewById<CheckBox>(R.id.checkboxVerboseIntegrationDetection).isChecked
+    val noMock = findViewById<RadioButton>(R.id.radioNoMock).isChecked
+    val allMocked = findViewById<RadioButton>(R.id.radioAllMocked).isChecked
+    val noMediation = findViewById<RadioButton>(R.id.radioNoMediation).isChecked
+
+    if (!verbose && noMock) {
```

Comment:
Because for a good UX, we should then disable the mock radio box when verbose is selected, and I don't like UI :D

More seriously, for testing the mediation adapters, we would need to select the mocking option. Verbosity can be selected too in order to show explicitly what's going on